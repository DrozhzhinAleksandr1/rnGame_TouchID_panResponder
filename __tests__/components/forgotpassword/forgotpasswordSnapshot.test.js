import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import {ForgotPasswordScreen}  from '../../../app/components/forgotpassword/forgotpassword';
import stackNav from '../../../app/redux/reducers/stacknavreducer';
import { localization } from '../../../app/common/localization';

stackNav.setParams = () => {
    return "test";
}

it('create snapshot ForgotPasswordScreen', () => {
    const renderer = new ShallowRenderer()
    let toolbarSnapshot = renderer.render(<ForgotPasswordScreen  
        navigation={stackNav}  
        CurrentLanguage={localization.ru}
        loginError={false}
        />);
    expect(toolbarSnapshot).toMatchSnapshot();
});
