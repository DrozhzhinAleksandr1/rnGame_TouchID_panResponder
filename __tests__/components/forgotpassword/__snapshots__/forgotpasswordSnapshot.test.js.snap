// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create snapshot ForgotPasswordScreen 1`] = `
<Component
  style={
    Object {
      "backgroundColor": "#172533",
      "height": "100%",
      "left": 0,
      "position": "absolute",
      "top": 0,
      "width": "100%",
    }
  }
>
  <withNavigation(Connect(ForgotPasswordPopUp)) />
  <ScrollView
    keyboardShouldPersistTaps="handled"
    style={
      Object {
        "height": "100%",
        "width": "100%",
      }
    }
  >
    <Component
      style={
        Object {
          "marginTop": "10%",
        }
      }
    >
      <Component
        style={
          Object {
            "alignItems": "center",
            "marginBottom": "10%",
          }
        }
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "#FFFFFF",
              "fontFamily": "Roboto-Regular",
              "fontSize": 20,
              "lineHeight": 23,
              "marginTop": 40,
            }
          }
        >
          Восстановление пароля
        </Text>
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "rgba(255, 255, 255, 0.6)",
              "fontFamily": "Roboto-Regular",
              "fontSize": 16,
              "lineHeight": 19,
              "marginBottom": 25,
              "marginTop": 5,
              "textAlign": "center",
              "width": 288,
            }
          }
        >
          Пожалуйста, укажите e-mail, который Вы использовали при регистрации
        </Text>
        <Input
          accessibilityLabel="ForgotPasswordInputID"
          error={false}
          errorText="Ошибка ввода e-mail"
          filter={/\\^\\(\\(\\[\\^<>\\(\\)\\\\\\[\\\\\\]\\\\\\\\\\.,;:\\\\s@"\\]\\+\\(\\\\\\.\\[\\^<>\\(\\)\\\\\\[\\\\\\]\\\\\\\\\\.,;:\\\\s@"\\]\\+\\)\\*\\)\\|\\("\\.\\+"\\)\\)@\\(\\(\\\\\\[\\[0-9\\]\\{1,3\\}\\\\\\.\\[0-9\\]\\{1,3\\}\\\\\\.\\[0-9\\]\\{1,3\\}\\\\\\.\\[0-9\\]\\{1,3\\}\\\\\\]\\)\\|\\(\\(\\[a-zA-Z\\\\-0-9\\]\\+\\\\\\.\\)\\+\\[a-zA-Z\\]\\{2,\\}\\)\\)\\$/}
          placeholder="Email"
          testID="ForgotPasswordInputID"
        />
        <FormBtnSubmit
          accessibilityLabel="ForgotPasswordSubmitID"
          func={[Function]}
          testID="ForgotPasswordSubmitID"
          text="Отправить"
        />
      </Component>
    </Component>
  </ScrollView>
  <Connect(ErrorValidation)
    testIDBtn="ForgotPasswordBtnId"
    testIDText="ForgotPasswordTextId"
  />
</Component>
`;
